tests/conftest.py

```
@@ -0,0 +1,44 @@
+"""Pytest configuration and fixtures."""
+
+import pytest
+import discord
```

Ruff (0.8.2)

`discord` imported but unused

Remove unused import: `discord`

(F401)